pkg/webhooks/validator/validator.go

```diff
@@ -168,13 +172,15 @@ func (wh *WebhookHandler) ValidateCreate(ctx context.Context, dryrun bool, hc *v
 	return nil
 }
 
-func (wh *WebhookHandler) getOperands(requested *v1beta1.HyperConverged) (*kubevirtcorev1.KubeVirt, *cdiv1beta1.CDI, *networkaddonsv1.NetworkAddonsConfig, error) {
+func (wh *WebhookHandler) getOperands(ctx context.Context, requested *v1beta1.HyperConverged) (*kubevirtcorev1.KubeVirt, *cdiv1beta1.CDI, *networkaddonsv1.NetworkAddonsConfig, error) {
 	if err := wh.validateCertConfig(requested); err != nil {
 		return nil, nil, nil, err
 	}
 
-	kv, err := operands.NewKubeVirt(requested)
-	if err != nil {
+	kv, err := operands.NewKubeVirt(ctx, wh.cli, requested)
+	if errT, ok := err.(*operands.RateTuningError); ok {
+		wh.logger.Error(errT, "failed to set the rate limit")
```

Comment:
Nit: can drop `errT` var. logger.Error will work with the existing `err` variable.

Reply:
Done.